include/zcbor_common.h

```diff
@@ -446,6 +476,31 @@ float zcbor_float16_to_32(uint16_t input);
  */
 uint16_t zcbor_float32_to_16(float input);
 
+#ifdef ZCBOR_MAP_SEARCH_FLAGS
+static inline size_t zcbor_round_up(size_t x, size_t align)
+{
+	return (((x) + (align) - 1) / (align) * (align));
+}
+
+/** Calculate the number of bytes needed to hold @p num_flags 1 bit flags
+ */
+static inline size_t zcbor_flags_to_bytes(size_t num_flags)
+{
+	return zcbor_round_up(num_flags, 8) / 8;
```

Comment:
nit: would this be easier to read as `return zcbor_round_up(num_flags, BITS_IN_BYTE) / BITS_IN_BYTE;` or similar?

Reply:
Done